### PR TITLE
[IMP] hr(_contract) : Add a graph and pivot view for employee

### PR DESCRIPTION
**Pull request by jugj-odoo**
In this PR we add a graph and pivot view that aggregates employees that were created / have their first contract date per month.
task-3374966

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr/views/hr_employee_views.xml [hunk 1]
```diff
@@ -239,6 +239,28 @@
...
+            <field name="arch" type="xml">
+                <graph type="line">
+                    <field name="create_date" interval="month"/>
+                    <field name="id"/>
+                </graph>
+            </field>
+        </record>
+
+        <record id="view_hr_employee_pivot" model="ir.ui.view">
+            <field name="name">hr.employee.pivot</field>
```
Comment (sofiagvaladze) [inline addons/hr/views/hr_employee_views.xml:254]: same naming convention here

File: addons/hr_contract/views/hr_employee_views.xml [hunk 2]
```diff
@@ -10,4 +10,28 @@
             </xpath>
         </field>
     </record>
+
+    <record id="view_hr_employee_graph_inherit" model="ir.ui.view">
+        <field name="name">hr.employee.graph</field>
```
Comment (sofiagvaladze) [inline addons/hr_contract/views/hr_employee_views.xml:15]: for inherited view  
`<record id="hr_employee_view_graph_inherit_hr_contract" model="ir.ui.view"> 
`
and we match name as well


https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html?highlight=naming#inheriting-xml

File: addons/hr_contract/views/hr_employee_views.xml [hunk 3]
```diff
@@ -10,4 +10,28 @@
...
+        <field name="priority">0</field>
+        <field name="arch" type="xml">
+            <field name="create_date" position="replace">
+                <field name="first_contract_date" interval="month"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="view_hr_employee_pivot_inherit" model="ir.ui.view">
+        <field name="name">hr.employee.pivot</field>
```
Comment (sofiagvaladze) [inline addons/hr_contract/views/hr_employee_views.xml:27]: same here

**Review by sofiagvaladze — CHANGES_REQUESTED** [review]
@jugj-odoo thanks for your work :raised_hands: 

I am nitpicking a bit on naming conventions here :innocent: It makes code readable and reviewers happy :bulb:

**Comment by kba-odoo** [comment]
robodoo r+

**Pull request closed without merging**